src/components/shared/Layout.tsx

```diff
@@ -50,20 +50,23 @@ function Layout(props: LayoutProps): JSX.Element {
   }, [sceneIdx]);
 
   useEffect(() => {
-    animateLeft('.foreground-image');
+    setTimeout( () => {
+      animateLeft('.foreground-image');
+    }, 3000);
   }, [scene]);
 
   useEffect(() => {
     if (expoIdx >= scene.expo.length) return;
-
-    animateUp(`.text-wrapper #expo-${expoIdx}`);
-    timeout.current && clearTimeout(timeout.current);
-    timeout.current = setTimeout(() => {
-      if (expoIdx === scene.expo.length - 1) showButton(nextRef);
-      animateOutUp(`.text-wrapper #expo-${expoIdx}`);
-      setExpoIdx(expoIdx + 1);
-      timeout.current = null;
-    }, (scene.expo[expoIdx].duration ?? 2000));
+    setTimeout( () => {
+      animateUp(`.text-wrapper #expo-${expoIdx}`);
+      timeout.current && clearTimeout(timeout.current);
+      timeout.current = setTimeout(() => {
+        if (expoIdx === scene.expo.length - 1) showButton(nextRef);
+        animateOutUp(`.text-wrapper #expo-${expoIdx}`);
+        setExpoIdx(expoIdx + 1);
+        timeout.current = null;
+      }, (scene.expo[expoIdx].duration ?? 2000));
+    }, 2000);
```

Comment:
the timer for these too dont match

Reply:
also u should test what happens if a user is on a page and then refreshes before/after this timeout expires and see what happens

Reply:
hmm i actually think the foreground some come in before the text.. at least when I was looking at the deploy preview, that was what made the most sense